config/config.go

```diff
@@ -368,6 +419,34 @@ type Log struct {
 	RecordPlanInSlowLog uint32     `toml:"record-plan-in-slow-log" json:"record-plan-in-slow-log"`
 }
 
+// Instance is the section of instance scope system variables.
+type Instance struct {
+	// These variables only exist in [instance] section.
+
+	// GeneralLog is used to log every query in the server in info level.
+	GeneralLog bool `toml:"tidb_general_log" json:"tidb_general_log"`
+	// EnablePProfSQLCPU is used to add label sql label to pprof result.
+	EnablePProfSQLCPU bool `toml:"tidb_pprof_sql_cpu" json:"tidb_pprof_sql_cpu"`
+	// DDLSlowOprThreshold sets log DDL operations whose execution time exceeds the threshold value.
+	DDLSlowOprThreshold uint32 `toml:"ddl_slow_threshold" json:"ddl_slow_threshold"`
+	// ExpensiveQueryTimeThreshold indicates the time threshold of expensive query.
+	ExpensiveQueryTimeThreshold uint64 `toml:"tidb_expensive_query_time_threshold" json:"tidb_expensive_query_time_threshold"`
+
+	// These variables exist in both 'instance' section and another place.
+	// The configuration in 'instance' section takes precedence.
+
+	EnableSlowLog         AtomicBool `toml:"tidb_enable_slow_log" json:"tidb_enable_slow_log"`
+	SlowThreshold         uint64     `toml:"tidb_slow_log_threshold" json:"tidb_slow_log_threshold"`
+	RecordPlanInSlowLog   uint32     `toml:"tidb_record_plan_in_slow_log" json:"tidb_record_plan_in_slow_log"`
+	CheckMb4ValueInUTF8   AtomicBool `toml:"tidb_check_mb4_value_in_utf8" json:"tidb_check_mb4_value_in_utf8"`
+	ForcePriority         string     `toml:"tidb_force_priority" json:"tidb_force_priority"`
+	MemoryUsageAlarmRatio float64    `toml:"tidb_memory_usage_alarm_ratio" json:"tidb_memory_usage_alarm_ratio"`
+	// EnableCollectExecutionInfo enables the TiDB to collect execution info.
+	EnableCollectExecutionInfo bool   `toml:"tidb_enable_collect_execution_info" json:"tidb_enable_collect_execution_info"`
+	PluginDir                  string `toml:"plugin_dir" json:"plugin_dir"`
```

Comment:
```suggestion
	TiDBPluginDir                  string `toml:"tidb_plugin_dir" json:"tidb_plugin_dir"`
```
Should we rename it to 'tidb_plugin_dir'?

Reply:
The behavior is identical to MySQL's plugindir option, so I don't think we need to rename it.